code/network/multiutil.cpp

```diff
@@ -3089,6 +3093,10 @@ void multi_update_valid_missions()
 		Multi_create_mission_list[idx].valid_status = (char)rval;
 	}
 
+	if ( !Is_standalone) {
```

Comment:
At some point we should standardize which way to check if the instance is a standalone.  `Game_mode & GM_STANDALONE_SERVER`  I have seen in more places.

Reply:
I did it that way at first and got a compiler error. Let me go back and check it again since I agree that we should standardize on the `Game_mode` check except in places that we can't/shouldn't include `freespace.h`.